<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCMAVLinkLogPlayer</class>
 <widget class="QWidget" name="QGCMAVLinkLogPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="logStatsLabel">
       <property name="text">
        <string>No logfile selected..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="logFileNameLabel">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="toolTip">
        <string>Start to replay the logfile</string>
       </property>
       <property name="statusTip">
        <string>Start to replay the logfile</string>
       </property>
       <property name="whatsThis">
        <string>Start to replay the logfile</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qgroundcontrol.qrc">
         <normaloff>:/files/images/actions/media-playback-start.svg</normaloff>:/files/images/actions/media-playback-start.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="positionLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="positionSlider">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="speedTitleLabel">
       <property name="toolTip">
        <string>Current replay speed</string>
       </property>
       <property name="statusTip">
        <string>Current replay speed</string>
       </property>
       <property name="whatsThis">
        <string>Current replay speed</string>
       </property>
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="speedButton75">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>0.75X</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton100">
         <property name="text">
          <string>1x</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton150">
         <property name="text">
          <string>1.5X</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton200">
         <property name="text">
          <string>2X</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton500">
         <property name="text">
          <string>5X</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="speedButton1000">
         <property name="text">
          <string>10X</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileButton">
       <property name="toolTip">
        <string>Select the logfile to replay</string>
       </property>
       <property name="statusTip">
        <string>Select the logfile to replay</string>
       </property>
       <property name="whatsThis">
        <string>Select the logfile to replay</string>
       </property>
       <property name="text">
        <string>Replay Logfile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qgroundcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
